packages/clarity-visualize/src/layout.ts

```diff
@@ -338,6 +338,15 @@ function setAttributes(node: HTMLElement, data: Layout.DomData): void {
 
     // Add an empty ALT tag on all IMG elements
     if (tag === Constant.ImageTag && !node.hasAttribute(Constant.AltAttribute)) { node.setAttribute(Constant.AltAttribute, Constant.Empty); }
+
+    // Prevent auto fill from working on forms during visualize. 
+    // During visualization, form fields are filled with saved details from the browser of user who is seeing the visualization
```

Comment:
Wording. Make it clear that autocomplete=off disables use of the saved information to autocomplete the form. It sounds like the opposite from this.